client/src/SharedComponents/Slideshow/Slideshow.js

```diff
@@ -8,9 +8,15 @@ class Slideshow extends Component {
   render() {
     const { projects, students, mentors } = this.props;
 
-    const projectSlide = projects.map((project) => {
-      return <ProjectView projectImg={project.project_image} />;
+    // Use new set proparity to remove redundent images of projects that comes from join query
+    const projectImages = projects.map((project) => {
+      return project.project_image;
     });
+    let uniqueProjects = [...new Set(projectImages)];
```

Comment:
Nice!
It might be useful in the future to put this into a utility function in another file (if you have to remove redundant array items again somewhere else).

Reply:
done! ,thanks for the note.